### PR TITLE
update to the agreed rules

### PR DESCRIPTION
**Pull request by FL-K**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -42,9 +42,9 @@ module.exports = {
             },
         ],
         '@typescript-eslint/no-explicit-any': 'error',
-        'no-tabs': ['off'],
-        'no-mixed-spaces-and-tabs': ['off'],
-        'no-undef': 'off',
+        'no-tabs': ['error'], // check no-tabs rule config to allow tabes in comments ?
+        'no-mixed-spaces-and-tabs': ['error'],
+        'no-undef': 'error',
```
Comment (stavros-liaskos) [inline index.js:47]: https://eslint.org/docs/rules/no-undef#:~:text=The%20%22extends%22%3A%20%22eslint%3Arecommended%22%20property%20in%20a%20configuration%20file%20enables%20this%20rule.

We should check the rest too

**Review by stavros-liaskos — CHANGES_REQUESTED** [review]

File: index.js (hunk 1 above)
Comment (FL-K) [reply index.js:47]: I am not sure I got it. To check for what ? You mean to turn some option off or to remove the comment in line 45
I can remove the comment I forgot it was just a fast note for after next steps to config it

Comment (stavros-liaskos) [reply index.js:47]: In the link attached it's highlighted:
`The "extends": "eslint:recommended" property in a configuration file enables this rule.`

So we don't need to add it manually. 
I assume the same goes for some other rules too. We don't need to duplicate them then

Comment (FL-K) [reply index.js:47]: Fair point yes we need to delete all rules that are already included in the recommended.
I am leaving this pr for reference only then to see what we agreed on and person that continue work here need to delete rules already included in recommended

**Pull request closed without merging**